cognee/api/v1/cognify/cognify_v2.py

```diff
@@ -87,39 +89,40 @@ async def run_cognify_pipeline(dataset: Dataset):
             cognee_config = get_cognify_config()
             graph_config = get_graph_config()
             root_node_id = None
-
-            if graph_config.infer_graph_topology and graph_config.graph_topology_task:
-                from cognee.modules.topology.topology import TopologyEngine
-                topology_engine = TopologyEngine(infer=graph_config.infer_graph_topology)
-                root_node_id = await topology_engine.add_graph_topology(files = data)
-            elif graph_config.infer_graph_topology and not graph_config.infer_graph_topology:
-                from cognee.modules.topology.topology import TopologyEngine
-                topology_engine = TopologyEngine(infer=graph_config.infer_graph_topology)
-                await topology_engine.add_graph_topology(graph_config.topology_file_path)
-            elif not graph_config.graph_topology_task:
-                root_node_id = "ROOT"
+            #
+            # if graph_config.infer_graph_topology and graph_config.graph_topology_task:
+            #     from cognee.modules.topology.topology import TopologyEngine
+            #     topology_engine = TopologyEngine(infer=graph_config.infer_graph_topology)
+            #     root_node_id = await topology_engine.add_graph_topology(files = data)
+            # elif graph_config.infer_graph_topology and not graph_config.infer_graph_topology:
+            #     from cognee.modules.topology.topology import TopologyEngine
+            #     topology_engine = TopologyEngine(infer=graph_config.infer_graph_topology)
+            #     await topology_engine.add_graph_topology(graph_config.topology_file_path)
+            # elif not graph_config.graph_topology_task:
+            #     root_node_id = "ROOT"
 
             tasks = [
-                Task(process_documents, parent_node_id = root_node_id), # Classify documents and save them as a nodes in graph db, extract text chunks based on the document type
-                Task(establish_graph_topology, topology_model = KnowledgeGraph, task_config = { "batch_size": 10 }), # Set the graph topology for the document chunk data
-                Task(expand_knowledge_graph, graph_model = KnowledgeGraph, collection_name = "entities"), # Generate knowledge graphs from the document chunks and attach it to chunk nodes
-                Task(filter_affected_chunks, collection_name = "chunks"), # Find all affected chunks, so we don't process unchanged chunks
+                Task(document_to_ontology, root_node_id = root_node_id),
+                Task(source_documents_to_chunks, parent_node_id = root_node_id), # Classify documents and save them as a nodes in graph db, extract text chunks based on the document type
+                Task(chunk_to_graph_decomposition_task, topology_model = KnowledgeGraph, task_config = { "batch_size": 10 }), # Set the graph topology for the document chunk data
```

Comment:
Some tasks don't have `_task` in their name and some have. I prefer not having it.

Reply:
done, fixed